### PR TITLE
release: v0.9.4-rc.1

### PR DESCRIPTION
**Pull request by darccio**
Upgrades dd-trace-go to `v1.69.1-rc.4`.

### CODE REVIEW

**Review by eliottness — CHANGES_REQUESTED** [review]
Can you add the support for jackc/pgx in the readme ? 👍

**Comment by darccio** [comment]
> Can you add the support for jackc/pgx in the readme ? 👍

@eliottness Done!

File: README.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ Library                               | Since    | Notes
 `github.com/redis/go-redis`           | `v0.9.1` | [Aspect][go-redis]
 `github.com/graph-gophers/graphql-go` | `v0.9.1` | [Aspect][graph-gophers]
 `github.com/graphql-go/graphql`       | `v0.9.1` | [Aspect][graphql]
+`github.com/jackc/pgx`                | `v0.9.4` | [Aspect][pgx]
```
Comment (eliottness) [inline README.md:152]: You need to add the link to the doc correctly at the end of the file 👍

**Review by eliottness — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (darccio) [reply README.md:152]: You are right, I left it behind accidentally.

image

**Review by eliottness — APPROVED** [review]

**Pull request merged**